bpython/importcompletion.py

```diff
@@ -190,19 +194,21 @@ def find_modules(path):
             continue
         else:
             if is_package:
-                for subname in find_modules(pathname):
-                    if subname != "__init__":
-                        yield "%s.%s" % (name, subname)
+                pathReal = os.path.realpath(pathname)
+                if not pathReal in paths:
+                    paths.add(pathReal)
+                    for subname in find_modules(pathname):
+                        if subname != "__init__":
+                            yield "%s.%s" % (name, subname)
             yield name
 
-
```

Comment:
the convention being followed here is two lines between top-level statements. You should `pip install black` and run black on this file; that will autoformat it, dealing with these details. Or better you can [set up VSCode to run black automatically](https://code.visualstudio.com/docs/python/editing#_formatting).